src/index.js

```diff
@@ -5,8 +5,18 @@ const TYPE_IMPORT = 'Import';
 
 const buildImport = template(`
   (new Promise((resolve) => {
-    require.ensure([], (require) => {
-      resolve(require(SOURCE));
+    require.ensure([SOURCE], (require) => {
+      var ns = require(SOURCE);
+      
+      if (ns && ns.__esModule) {
+        resolve(ns);
+      } else {
+        var wrap = Object.create(null);
+        wrap.__esModule = true;
+        wrap.default = ns;
```

Comment:
https://tc39.github.io/ecma262/#sec-module-namespace-exotic-objects-getownproperty-p enumerable is always true